diff --git a/setup.cfg b/setup.cfg
index 474df56..222d6a6 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -2,7 +2,7 @@
 name = cbor2
 description = Pure Python CBOR (de)serializer with extensive tag support
 long_description = file: README.rst
-author = Alex Grönholm
+author = Alex Gronholm
 author_email = alex.gronholm@nextday.fi
 project_urls =
     Documentation = https://cbor2.readthedocs.org/en/latest/
